pagopa/io-app
# Fix missing translations in english for Operazione successful!

## Short description
Fixes the incorrect toast message displayed when archiving a message on the Home page. Previously, users would see a mixed-language message: "Operazione successful!"

## List of changes proposed in this pull request
- [x] Updated the archive success message to show "Operation successful" instead of "Operazione successful!"
- [x] Applied linting to the translation file

## How to test

1. Set the device language to English.
2. Run the production app.
3. On the Home page, attempt to archive a message.
4. Verify that the toast message displays "Operation successful" in English only.
